modules/claims_api/lib/claims_api/v2/disability_compensation_pdf_mapper.rb

```diff
@@ -652,7 +652,8 @@ def national_guard # rubocop:disable Metrics/MethodLength
 
           area_code = reserves&.dig(:unitPhone, :areaCode)
           phone_number = reserves&.dig(:unitPhone, :phoneNumber)
-          reserves[:unitPhoneNumber] = convert_phone(area_code + phone_number) if area_code && phone_number
+          phone_number&.delete! '-'
+          reserves[:unitPhoneNumber] = (area_code + phone_number) if area_code && phone_number
```

Comment:
Just realizing this now, and I understand it is in already existing code, but I want to ask, do we want this to be blank if they only give a phone number without area code or the other way around (I understand area code by itself is pretty silly)?  Looking at the schema, since both are nullable, I would think if they provide a phone Number we would want it on the PDF?  Maybe a question for @klwatanabe

Reply:
Probably better question for stand up or something, everything here looks good.